suite-common/connect-init/src/lockDeviceConnectPatch.ts

```diff
@@ -0,0 +1,39 @@
+import TrezorConnect from '@trezor/connect';
```

Comment:
Can you move this stuff back to `prepareConnectInit` and maybe even get rid of `lockDeviceConnectPatch ` function and use it's code directly in thunk (with dispatches instead of callbacks etc.)

Reply:
https://github.com/trezor/trezor-suite/pull/5760/commits/979f44f2e30665c851548bc75ae31ae9d9bbba9b